src/ir/auth_logic/souffle_emitter_test.cc

```diff
@@ -42,220 +42,241 @@ Program BuildPredicateTestProgram() {
   EXPECT_EQ(actual, expected);
 }
 
-Program BuildMultiAssertionProgram() {
-  // Conditional Assertion
-  std::vector<BaseFact> rhs = {BaseFact(
-      datalog::Predicate("isEmployee", {"somePerson"}, datalog::kPositive))};
-  Fact lhs({},
-           BaseFact(datalog::Predicate("canAccess", {"somePerson", "someFile"},
-                                       datalog::kPositive)));
-  SaysAssertion condAssertion = BuildSingleSaysAssertion(
-      Principal("TestPrincipal"),
-      Assertion(ConditionalAssertion(std::move(lhs), std::move(rhs))));
-
-  // Assertion stating the condition
-  SaysAssertion predicateAssertion = BuildSingleSaysAssertion(
-      Principal("TestPrincipal"),
-      Assertion(
-          Fact({}, BaseFact(datalog::Predicate("isEmployee", {"somePerson"},
-                                               datalog::kPositive)))));
-
-  // I would love to just write this:
-  // return Program({std::move(condAssertion),
-  //     std::move(predicateAssertion)}, {});
-
-  std::vector<SaysAssertion> assertion_list = {};
-  assertion_list.push_back(std::move(condAssertion));
-  assertion_list.push_back(std::move(predicateAssertion));
-  return Program({}, std::move(assertion_list), {});
-}
-
-TEST(EmitterTestSuite, MultiAssertionProgramTest) {
-  std::string expected =
-      R"(says_canAccess(TestPrincipal, somePerson, someFile) :- says_isEmployee(TestPrincipal, somePerson).
-says_isEmployee(TestPrincipal, somePerson).
-grounded_dummy(dummy_var).
-
-.decl grounded_dummy(x0: symbol)
-.decl says_canAccess(x0: symbol, x1: symbol, x2: symbol)
-.decl says_isEmployee(x0: symbol, x1: symbol)
-)";
-  std::string actual = SouffleEmitter::EmitProgram(
-      LoweringToDatalogPass::Lower(BuildMultiAssertionProgram()));
-  EXPECT_EQ(actual, expected);
-}
-
-Program BuildQueryProgram() {
-  Query testQuery("theTestQuery", Principal("TestSpeaker"),
-                  Fact({}, BaseFact(datalog::Predicate("anything", {"atAll"},
-                                                       datalog::kPositive))));
-  std::vector<Query> query_list = {};
-  query_list.push_back(std::move(testQuery));
-  return Program({}, {}, std::move(query_list));
-}
-
-TEST(EmitterTestSuite, QueryTestProgram) {
-  std::string expected =
-      R"(theTestQuery(dummy_var) :- says_anything(TestSpeaker, atAll), grounded_dummy(dummy_var).
-grounded_dummy(dummy_var).
-.output theTestQuery
-.decl grounded_dummy(x0: symbol)
-.decl says_anything(x0: symbol, x1: symbol)
-.decl theTestQuery(x0: symbol)
-)";
-
-  // R"(theTestQuery(dummy_var) :- says_anything(TestSpeaker, atAll),
-  // grounded_dummy(dummy_var). grounded_dummy(dummy_var). .output theTestQuery
-  //
-  // .decl theTestQuery(x0: symbol)
-  // .decl says_anything(x0: symbol, x1: symbol)
-  // .decl grounded_dummy(x0: symbol))";
-
-  std::string actual = SouffleEmitter::EmitProgram(
-      LoweringToDatalogPass::Lower(BuildQueryProgram()));
-  EXPECT_EQ(actual, expected);
-}
+// Program BuildAttributeTestProgram() {
```

Comment:
Is there a reason to include this commented out code in the repository? If not, it's probably better to delete this rather than comment it out. If for some reason we need to refer back to this commented out code, it's still in the git revision history. (Just make a note of the commit where this code was last in the repo so that you can find it again).

Reply:
Removed commented out section!